INSTALL.md

```diff
@@ -1 +1,137 @@
-# TODO
+# Building opentelemetry-cpp
+
+[Bazel](https://bazel.build) and [CMake](https://cmake.org/) are the official
+build systems for opentelemetry-cpp.
+
+## Build instructions using CMake
+
+### Prerequisites
+
+- A supported platform (eg Windows, macOS, Linux etc).
+Refer [Platforms Supported](./README.md#supported-development-platforms)
+for more information.
+- A compatible C++ compiler supporting atleast C++11.
+Major compilers are supported.
+Refer [Supported Compilers](./README.md#supported-c-versions) for more information.
+- [Git](https://git-scm.com/) for fetching opentelemetry-cpp source code from repository.
+To install Git, consult the [Set up Git](https://help.github.com/articles/set-up-git/)
+guide on GitHub.
+- [CMake](https://cmake.org/) for building opentelemetry-cpp API,
+SDK with their unittests. We use CMake version 3.15.2 in our build system.
+To install CMake, consult the [Installing CMake](https://cmake.org/install/) guide.
+- [GoogleTest](https://github.com/google/googletest) framework to build and
+run the unittests.
+We use GoogleTest version 1.10.0 in our build system. To install GoogleTest,
+consult the [GoogleTest Build Instructions](https://github.com/google/googletest/blob/master/googletest/README.md#generic-build-instructions).
+- Apart from above core requirements, the Exporters and Propagators have their
+build dependencies which are not covered here. E.g, Otlp Exporter needs
+grpc/protobuf library, Zipkin exporter needs nlohmann-json and libcurl,
+ETW exporter need nlohmann-json to build. This is covered in the build
+instructions for these components.
+
+### Building as Standalone CMake Project
+
+1. Getting the opentelementry-cpp source:
+
+   ```console
+   # Change to the directory where you want to create the code repository
+   $ cd ~
+   $ mkdir source && cd source
+   $ git clone https://github.com/open-telemetry/opentelemetry-cpp
```

Comment:
Add option `--recursive` to clone submodule as well?

Reply:
submodules needn't be cloned for api/sdk build. It would be required for exporters, but better to have separate instructions for them in their respective folders.